presto-bigquery/src/test/java/io/prestosql/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -105,6 +105,33 @@ public void testPredicatePushdownPrunnedColumns()
                 "VALUES (1)");
     }
 
+    @Test(description = "regression test for https://github.com/prestosql/presto/issues/5635")
+    public void testCountAggregationView()
+    {
+        BigQuery client = createBigQueryClient();
+
+        String tableName = "test.count_aggregation_table";
+        String viewName = "test.count_aggregation_view";
+
+        executeBigQuerySql(client, "DROP TABLE IF EXISTS " + tableName);
+        executeBigQuerySql(client, "DROP VIEW IF EXISTS " + viewName);
+        executeBigQuerySql(client, "CREATE TABLE " + tableName + " (a INT64, b INT64, c INT64)");
+        executeBigQuerySql(client, "INSERT INTO " + tableName + " VALUES (1, 2, 3)");
```

Comment:
Can you please add two rows to a table, so the expected `count(*)` with and without filters is different?